internal/vds/vds.go

```diff
@@ -79,6 +85,21 @@ func GetAxis(direction string) (int, error) {
 	}
 }
 
+func GetCoordinateSystem(coordinateSystem string) (int, error) {
+	switch strings.ToLower(coordinateSystem) {
+	case "ij":
+		return CoordinateSystemIJ, nil
+	case "ilxl":
+		return CoordinateSystemIlXl, nil
+	case "cdp":
+		return CoordinateSystemCdp, nil
+	default:
+		options := "ij, ilxl, cdp"
+		msg := "coordinate system not recognized: '%s', valid options are: %s"
+		return 0, fmt.Errorf(msg, coordinateSystem, options)
```

Comment:
As you pointed out in Eirik's pr, returning -1 is a better choice here. GetAxis arguably does the wrong thing. Maybe we can get a tiny fixup commit for that one too ? 😇

Reply:
Ah, I didn't even notice :D